packages/suite/src/actions/wallet/coinjoinClientActions.ts

```diff
@@ -76,6 +77,12 @@ const clientOnStatusEvent = (symbol: Account['symbol'], status: CoinjoinStatusEv
         },
     } as const);
 
+const clientOnPrisonEvent = (event: CoinjoinClientEvents['prison']) =>
+    ({
+        type: COINJOIN.CLIENT_PRISON_EVENT,
```

Comment:
note to myself: check if it makes sense to trigger save to storage or waiting for account update is enough

Reply:
@szymonlesisz I believe it makes sense to store it on prison event too.

Reply:
[eaca752](https://github.com/trezor/trezor-suite/pull/8549/commits/eaca7520e018d800c866cd0fb2feeea7bcd862dd)